monai/fl/utils/dummy_filter.py

```diff
@@ -0,0 +1,23 @@
+from monai.fl.utils.exchange_object import ExchangeObject
+from monai.fl.utils.filter import Filter
+
+
+class DummyFilter(Filter):
```

Comment:
I'd suggest the class name `IdentityFilter` or `PrintingFilter` to give some information about what the class do

Reply:
Good point. I went for `SummaryFilter`.